test_files/decorator/external.js

```diff
@@ -17,3 +17,5 @@ exports.AClassWithGenerics = AClassWithGenerics;
  */
 function AType() { }
 exports.AType = AType;
+function AType_tsickle_Closure_declarations() {
+}
```

Comment:
Do we care about these empty functions?  Could be solved with an extra 'if' in the code.